src/Cargo.lock

```diff
@@ -265,6 +265,15 @@ name = "cfg-if"
 version = "0.1.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 
+[[package]]
+name = "chrono"
+version = "0.4.0"
```

Comment:
Could you *not* upgrade `env_logger`, to avoid conflict with #47501 (which is currently blocked due to `chrono 0.4.0` prevents compiler doc from being warning-free)?